### ISSUE TITLE
bug([COMPONENT]): [TITLE]

### ISSUE THREAD

**Issue by jimkeecn** [angular/components#18805]
#### Reproduction

I have create a github repo for this performance issue from angular material, here is the link https://github.com/jimkeecn/buildxactDemo

the problem in the repo is. I have created an angular element with a mat-autocomplete and mat-table. after build the angular element and used in other project. the mat-input is buggy, the placeholder will not move up while the input is focus and filled. Also the pagination of the mat-table is not working at the very start until i have trigger some detect change, however the pagination has slow reaction after click and werid styling and flicking of texts.

Steps to reproduce:
1. download the repo.
2. open the index.html in folder /I am outside of the project/
3. you will see the performance of the angular material input and table.
4. the component is actually in library project folder /search-page
5. angular element is in external project folder /app-component

 
 
#### Expected Behavior

I was expecting to see the mat-input will run smooth like it running without building as angular elements.
I was expecting the mat-table pagination is working, and pagination button is clickable.

#### Actual Behavior

mat-input
placeholder is not lifted up after the field is focus or filled.
pagination forward and backward button is not clickable 
pagination dropdown is not clickable

#### Environment

- Angular: ~9.0.3
- Angular/elements: ^9.0.4
- CDK/Material: ^9.1.0
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu):  Windows 10

**Comment by jimkeecn** [comment]
@Splaktar this is the issue I mentioned.

**Comment by lucaboy** [comment]
I'm facing the same issues (Angular 9.0.5 + CDK/Material: 9.1.2).

**Comment by Splaktar** [comment]
Updated the title. If you also have an issue with MatTable in an Angular Element, a separate issue would be appreciated. We don't want to get multiple issues lost in a single report.

**Comment by lucaboy** [comment]
Disabling Ivy seems to solve the problem (edit the file _tsconfig.json_ and add the line _"enableIvy": false_ to angularCompilerOptions).
`  "angularCompilerOptions": {
    "fullTemplateTypeCheck": true,
    "strictInjectionParameters": true,
    "enableIvy": false,
  }`

**Comment by jimkeecn** [comment]
any update on this one??

**Comment by Splaktar** [comment]
@jimkeecn no, sorry. It hasn't been triaged yet, so I'm not sure what priority it will have.

**Comment by jimkeecn** [comment]
@Splaktar No problem

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
